src/Microsoft.AspNet.JsonPatch/Adapters/IObjectAdapter.cs

```diff
@@ -8,14 +8,14 @@ namespace Microsoft.AspNet.JsonPatch.Adapters
     /// <summary>
     /// Defines the operations that can be performed on a JSON patch document.
     /// </summary>
-    /// <typeparam name="T"></typeparam>
-    public interface IObjectAdapter<T> where T : class
+    /// <typeparam name="TModel">The type of the model.</typeparam>
+    public interface IObjectAdapter<TModel> where TModel : class
     {
-        void Add(Operation<T> operation, T objectToApplyTo);
-        void Copy(Operation<T> operation, T objectToApplyTo);
-        void Move(Operation<T> operation, T objectToApplyTo);
-        void Remove(Operation<T> operation, T objectToApplyTo);
-        void Replace(Operation<T> operation, T objectToApplyTo);
-        void Test(Operation<T> operation, T objectToApplyTo);
+        void Add(Operation<TModel> operation, TModel objectToApplyTo);
+        void Copy(Operation<TModel> operation, TModel objectToApplyTo);
+        void Move(Operation<TModel> operation, TModel objectToApplyTo);
+        void Remove(Operation<TModel> operation, TModel objectToApplyTo);
+        void Replace(Operation<TModel> operation, TModel objectToApplyTo);
+        void Test(Operation<TModel> operation, TModel objectToApplyTo);
```

Comment:
I kept `Test` API but just removed the code in `ObjectAdapter` and added an error that says "test operation is not supported"